File: JavaScript/1015 - Distância Entre Dois Pontos.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
var [x1, y1] = input[0].split(' ').map(parseFloat);
var [x2, y2] = input[1].split(' ').map(parseFloat);

var distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));
console.log(distancia.toFixed(4));

File: Clojure/1015 - Distância Entre Dois Pontos.clj
(let [linha1 (map #(Double/parseDouble %) (clojure.string/split (read-line) #"\s+"))
      linha2 (map #(Double/parseDouble %) (clojure.string/split (read-line) #"\s+"))
      x1 (nth linha1 0)
      y1 (nth linha1 1)
      x2 (nth linha2 0)
      y2 (nth linha2 1)
      distancia (Math/sqrt (+ (Math/pow (- x2 x1) 2)
                               (Math/pow (- y2 y1) 2)))]
  (println (format "%.4f" distancia)))